src/purview.ts

```diff
@@ -414,7 +248,7 @@ function makeStateTree(
   const childMap: ChildMap<StateTree> = {}
   Object.keys(component._childMap).forEach(key => {
     const children = component._childMap[key]!
-    childMap[key] = children.map(c =>
+    childMap[key] = children.map((c: any) =>
```

Comment:
The typechecker complained about "no implicit any", so I complied. Open to other ideas here or if pushback if there is something wrong with my environment.